Services/UI/OJS.Services.Ui.Data/Implementations/SubmissionsForProcessingDataService.cs

```diff
@@ -86,10 +121,28 @@ public async Task ResetProcessingStatusById(int id)
             {
                 submissionForProcessing.Processing = false;
                 submissionForProcessing.Processed = false;
+                submissionForProcessing.CreatedOn = DateTime.Now.ToUniversalTime();
+
                 await this.SaveChanges();
             }
         }
 
+        public async Task MarkProcessed(int submissionId)
```

Comment:
This seems the same as `EndProcessingBySubmission(int submissionId)`

Reply:
Yes, removed it